x-pack/plugins/cases/server/services/cases/types.ts

```diff
@@ -20,13 +20,25 @@ export type ExternalServicesWithoutConnectorId = Omit<
   'connector_id'
 >;
 
+export enum ESCaseSeverity {
+  LOW = 0,
+  MEDIUM = 1,
+  HIGH = 2,
+  CRITICAL = 3,
```

Comment:
I think we should change this to `0, 10, 20, 30`. This will give us the ability to put other severity values in the middle in the future if we need it. For example `SEVERE = 25`